integration/fuse/src/main/java/alluxio/fuse/file/FileStatus.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.file;
+
+/**
+ * The file status of ongoing fuse stream.
+ */
+public class FileStatus {
+  private long mFileLength;
```

Comment:
Only one field?

Reply:
Yeah, for read now only one field is needed. This is added to simplify the FuseFileStream interface